packages/graphql-language-service-types/package.json

```
@@ -14,21 +14,23 @@
   },
   "license": "MIT",
   "files": [
-    "dist"
+    "dist",
+    "esm"
   ],
   "keywords": [
     "graphql"
   ],
   "main": "dist/index.js",
-  "typings": "src/index",
+  "module": "esm/index.js",
+  "typings": "src/index.ts",
```

We're not bundling `src`.

Not sure if this should be `dist` or `esm`, but you put `esm` everywhere else so I'm going to trust that.

```suggestion
  "typings": "esm/index.d.ts",
```

---

Yeah that sounds like a build-order error to me. We can't set `typings` to be `src/*` because it won't be in the final bundle.

---

yep, the cross-references aren't completely set up

---

now that I added the one missing downstream project reference resolution it works properly at `dist/index.d.ts` :D